### PR TITLE
strip lock enhancements

### PR DESCRIPTION
**Pull request by fabrizzio-dotCMS**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/concurrent/lock/StripedLockImpl.java [hunk 1]
```diff
@@ -94,16 +106,45 @@ public void tryLock(final K key, final VoidDelegate callback) throws Throwable {
...
+            }
+            try {
+                callback.execute();
+            } finally {
+                lock.unlock();
+            }
         }
     }
+
+    private synchronized Thread getOwnerThread(final ReentrantLock lock) throws Exception{
```
Comment (wezell) [inline dotCMS/src/main/java/com/dotcms/concurrent/lock/StripedLockImpl.java]: why synchronized?

Comment (fabrizzio-dotCMS) [reply dotCMS/src/main/java/com/dotcms/concurrent/lock/StripedLockImpl.java]: yeah no need. I just changed that.

**Review by wezell — APPROVED** [review]
Looks good.

**Review by jdotcms — APPROVED** [review]
I would like to use cache provider as a map, but it is ok

**Review by jgambarios — APPROVED** [review]

**Pull request merged**